assets/src/edit-story/components/colorPicker/currentColorPicker.js

```diff
@@ -173,6 +226,17 @@ function CurrentColorPicker({ rgb, hsl, hsv, hex, onChange, showOpacity }) {
       </Body>
       <Footer>
         <HexValue>
+          <Button
+            variant={BUTTON_VARIANTS.SQUARE}
+            type={BUTTON_TYPES.QUATERNARY}
+            size={BUTTON_SIZES.SMALL}
+            aria-label={__('Pick a color from canvas', 'web-stories')}
+            onClick={initEyedropper}
+            onMouseEnter={prepareEyedropper}
```

Comment:
Should this happen only when entering the eyedropper the first time / after a new colour change from the picker?